### ISSUE TITLE
Inconsistency in page<>component layouts and spacing

### ISSUE THREAD

**Issue by amirhouieh** [acid-info/logos-docusaurus-plugins#74]
@nireims @jinhojang6 looking at pages [like this](https://dev-docs.waku.org/guides/run-nwaku-node/), you notice inconsistency in `width` and `spacing` in the layout. 

- [ ] We need to unified all components width to fit into the content container. 
- [ ] We need to unify all vertical spacing between component (unless in content there has been extra breakpoints) 

![Screenshot 2023-06-22 at 11 19 40](https://github.com/acid-info/logos-docusaurus-plugins/assets/7159454/0e034ef6-5dae-47d9-ad08-998c3952871c)
![Screenshot 2023-06-22 at 11 25 11](https://github.com/acid-info/logos-docusaurus-plugins/assets/7159454/f932e6fb-c78a-46bd-b5ea-eb663d61c4b3)
![Screenshot 2023-06-22 at 11 25 14](https://github.com/acid-info/logos-docusaurus-plugins/assets/7159454/82fb7055-40ec-4a32-a6de-6375a9aaf870)

**Comment by nireims** [comment]
Spacing between doc sections should be 32px and between elements in the section - 24px
Spacing from H1 to the content - 40px
Does it make sense? @amirhouieh 
![image](https://github.com/acid-info/logos-docusaurus-plugins/assets/115178918/23d96fc9-8086-4d05-90d9-a82f3d9095af)
![image](https://github.com/acid-info/logos-docusaurus-plugins/assets/115178918/10a3c14b-4bbc-48c9-9cdb-d9343589b4f6)

**Comment by nireims** [comment]
yes the spacing between text and any components in a section should be 24px

**Comment by amirhouieh** [comment]
@jinhojang6 plz proceed with this.

**Comment by jinhojang6** [comment]
https://github.com/acid-info/logos-docusaurus-plugins/pull/85

**Comment by jinhojang6** [comment]
PR ready for review
